summary_workflows/quantification/corr_with_gt.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.stats import pearsonr
+import json
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('bed', help='The BED file containing predictions merged with ground truth.')
+args = parser.parse_args()
+
+fname = args.bed
+out = pd.read_csv(fname, delimiter='\t', header=None)
+
+# initialize vectors for ground truth sites and prediction
+vec_true = []
+vec_pred = []
+
+# multiple predicted sites for one ground truth?
+multiple_predicted_sites = out.duplicated([6, 7, 8, 11], keep=False)
```

Comment:
I think it was mentioned already, but for better readability it would be nice to have named columns.